src/mock-data/fiat-account.ts

```diff
@@ -28,19 +28,19 @@ const ibanNumberAustria: PostFiatAccountRequestBody<FiatAccountSchema.IBANNumber
     },
   }
 
-const ibanNumberXOF: PostFiatAccountRequestBody<FiatAccountSchema.IBANNumber> =
+const accountNumberXOF: PostFiatAccountRequestBody<FiatAccountSchema.AccountNumber> =
```

Comment:
in general, I'd caution against updating existing fixture data, since it could be used by other providers' tests. Safer to add a new mock account number instead. However, in this case, I think Dunia was the only one using ibanNumberXOF, so it's probly fine to just change it.